src/openvisuspy/slice.py

```diff
@@ -269,64 +270,57 @@ def showData(self, data, viewport,color_bar=None):
 
 # ////////////////////////////////////////////////////////////////////////////////////
 class Slice(param.Parameterized):
+	def __init__(self): # just so that we can get new instances in each session

```

Comment:
Are you sure you have to do this? I thought the `param` library was automatically creating instance sessions automatically. We need to dig into it.

Reply:
It was not creating new instances earlier. Tried fixing the main.py file, and it wasnot working..  atleast I didnot find any other ways to do it except this.